wire-grpc-server/src/main/java/com/squareup/wire/GrpcChannel.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire
+
+import java.io.Closeable
+
+interface GrpcSendChannel<T> : Closeable {
```

Comment:
Should these go in grpc-common to give us the option for blocking clients?

Reply:
Why not? We'd have to support it in the runtime as well. Our `GrpcClient` assumes coroutines to create the service.

You'd add something like in the plugin config?
```kotlin
    var blockingServices: Boolean
    var singleMethodServices: Boolean
/!\ var kotlinCoroutine: Boolean /!\
```

Reply:
I think it's just blockingServices=true to use these.

Reply:
`blockingServices` makes it so we don't use `suspend` for a single request/response API. Others API are non-blocking by default.
Why would `blockingServices=true` affect the non-blocking APIs signature?

Reply:
I think there's two independent features:

 * blocking/suspending
 * single interface/many interfaces

We could build blocking clients and suspending servers. Just don't need those yet.

Reply:
Ok, then maybe we don't need the `wire-grpc-server` for now. I'll move the GrpcChannels into our `wire-grpc` module.